jobs/pipelines/powervs/ocp-4.8/ocp4.8-longevity-powervs-saopaulo01/Jenkinsfile

```diff
@@ -0,0 +1,204 @@
+                        }
+                        else{
+                            echo "${BASTION_IMAGE_FILE} file does not exist. Please check poll-powervs-job"
+                            throw err
+                        }
+                        if (fileExists("deploy/artifactory/${RHCOS_IMAGE_FILE}")) {
+                            env.RHCOS_IMAGE_NAME = readFile "deploy/artifactory/${RHCOS_IMAGE_FILE}"
+                            env.RHCOS_IMAGE_NAME = env.RHCOS_IMAGE_NAME.trim()
+                        }
+                        else{
+                            echo "${RHCOS_IMAGE_FILE} file does not exist. Please check poll-powervs-job"
+                            throw err
+                        }
+                        env.OPENSHIFT_INSTALL_TARBALL=getOpenshiftBuild(OCP_RELEASE)
+                        env.OPENSHIFT_CLIENT_TARBALL=getOpenshiftClient(OCP_RELEASE)
+                        env.OPENSHIFT_CLIENT_TARBALL_AMD64=getOpenshiftClientAMD(OCP_RELEASE)
+                    }
+                    catch (err)
+                    {
+                        echo 'Error ! Template preparation failed !'
+                        throw err
+                    }
+                }
+            }
+        }
+        stage('Initialize Environment') {
+            steps {
+                setupTerraformPlugin()
+                initializeEnvironment()
+            }
+        }
+        stage('Deploy OCP Cluster') {
+            steps {
+                deployCluster()
+            }
+        }
+        stage('Run crontab script for capturing outputs of multiple commands') {
+            steps {
+                crontabCommandCaptureScript()
+            }
+        }
+        stage('Setup Kubectl') {
+            steps {
+                setupKubeconfigOcp4()
+            }
+        }
+        stage('Longevity test - collect cronjob log and monitor the cluster with cerberus') {
+            steps {
+                parallel(
+                    a: {
+                       script {
+                           ansiColor('xterm') {
+                            echo ""
+                            }
+                            try {
+                                sh '''
+                                   k=0
+                                   echo "---SHOW CRON LOGS FROM LONGEVITY TEST - ONCE EVERY 8 hours / FOR 4 WEEKS---"
+                                   while [ $(k) -le 42 ]; do
+                                   sleep 28800
+                                   cd ${WORKSPACE}/deploy
+                                   scp -i id_rsa -o StrictHostKeyChecking=no root@${BASTION_IP}:~/cron.log cron_longevity.log
+                                   echo "---SHOWING  CRON LOGS---"
+                                   cat cron_longevity.log
+                                   k=$((k+1))
+                                   echo $k
+                                done
```

Comment:
lets do this log collection as part of capture-artifacts-ocp4.sh and add to artefacts

Reply:
@sudeeshjohn It's a part of archive artifacts. Actually Julie wanted these logs to print on console as this job is going to run for 3 weeks. So this step is added to pull that file every 8 hours and print in console logs

Reply:
Not creating a separate method for this as it might change

Reply:
is it possible to create this as a script ?

Reply:
@sudeeshjohn Created a method only. It might require changes in future